conf/declaration.routes

```diff
@@ -51,6 +51,12 @@ GET         /person-presenting-goods             controllers.declaration.PersonP
 
 POST        /person-presenting-goods             controllers.declaration.PersonPresentingGoodsDetailsController.submitForm(mode: models.Mode ?= models.Mode.Normal)
 
+# Exporter Eori
+
+GET         /exporter-eori-number                controllers.declaration.ExporterEoriNumberController.displayPage(mode: models.Mode ?= models.Mode.Normal)
+
+POST        /exporter-eori-number                controllers.declaration.ExporterEoriNumberController.submit(mode: models.Mode ?= models.Mode.Normal)
+
 # Exporter address
 
 GET         /exporter-details                    controllers.declaration.ExporterDetailsController.displayPage(mode: models.Mode ?= models.Mode.Normal)
```

Comment:
For consistency with other similar pages, could you change these two endpoints to `/exporter-address` ?

Reply:
Done